packages/gatsby-plugin-sharp/src/__tests__/process-file.js

```diff
@@ -94,5 +94,28 @@ describe(`createArgsDigest`, () => {
       testHashEqual(`maxWidth`, { maxWidth: 500 })
       testHashEqual(`base64`, { base64: true })
     })
+
+    describe(`argument sorting`, () => {
+      it(`sorts nested arguments`, () => {
+        // take the defaultargsbaseline, add a duotone argument (out of order)
+        const args = {
+          duotone: {
+            highlight: `#32CD32`,
+            shadow: `#10c5f8`,
```

Comment:
For sake of more coverage - could we maybe swap order of `shadow` and `highlight` to make sure nested objects have sorted keys as well?